core/field_angle.js

```diff
@@ -93,14 +93,36 @@ Blockly.FieldAngle.OFFSET = 0;
  */
 Blockly.FieldAngle.WRAP = 360;
 
-
 /**
  * Radius of protractor circle.  Slightly smaller than protractor size since
  * otherwise SVG crops off half the border at the edges.
  */
 Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
 
 /**
+ * Adds degree symbol and recalculates width.
+ * Saves the computed width in a property.
+ * @private
+ */
+Blockly.FieldAngle.prototype.render_ = function() {
+  if (!this.visible_) {
+    this.size_.width = 0;
+    return;
+  }
+
+  // Update textElement.
+  this.textElement_.innerHTML = this.getDisplayText_();
```

Comment:
Good point. I just read in the links you sent me that textContent is preferable and safer if we are talking about XSS attacks.

Source:
https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent